base/applications/osk/main.c

```diff
@@ -785,6 +797,82 @@ LRESULT OSK_Paint(HWND hwnd)
 
     return 0;
 }
+
+/***********************************************************************
+ *
+ *           OSK_ThemeHandler
+ *
+ *  Function helper which handles theme drawing of buttons with icons.
+ */
+LRESULT APIENTRY OSK_ThemeHandler(LPNMCUSTOMDRAW pNmDraw)
+{
+    HTHEME hTheme;
+    HWND hButton;
+    INT iState = PBS_NORMAL;
+    HICON hIcon;
+    int x, y;
+
+    hButton = pNmDraw->hdr.hwndFrom;
+
+    /* If the button doesn't have an icon there is no need for custom handling */
+    if (!(GetWindowLongW(hButton, GWL_STYLE) & BS_ICON))
+        return CDRF_DODEFAULT;
+
+    /* Retrieve the theme handle for the button controls */
+    hTheme = GetWindowTheme(hButton);
+
+    if (!hTheme)
+        return CDRF_DODEFAULT;
+
+    /* Obtain CDDS drawing stages */
+    switch (pNmDraw->dwDrawStage)
+    {
+        case CDDS_PREPAINT:
+        {
+            /*
+                The button could be either in normal state or pushed.
+                Retrieve its state and save to a variable.
+            */
```

Comment:
```suggestion
            /*
             * The button could be either in normal state or pushed.
             * Retrieve its state and save to a variable.
             */
```

Reply:
One of the many reasons why people should just generally stick with C++ comments // instead of C-style /* */.